### ISSUE TITLE
Using my username ends with an error

### ISSUE THREAD

**Issue by gillstrom** [SamVerschueren/dev-time#1]
``` js
devTime('gillstrom')
    .then(console.log)
    .catch(console.error);
```

Ends with the error below:

```
Error: Command failed: git log -1 --format="%aI" a48ada0d6939c14c9555551a3ff850e17be41cd1 fatal: bad object a48ada0d6939c14c9555551a3ff850e17be41cd1
```

Using a username like `kevva` works fine. My guess is this is happening because I [deleted the branch containing the commit](https://help.github.com/articles/commit-exists-on-github-but-not-in-my-local-clone/).

**Comment by SamVerschueren** [comment]
Thanks for reporting. This is indeed an edge case I didn't test :).

**Comment by SamVerschueren** [comment]
This should be fixed in `v0.2.1`.
